index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>home task 2</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body style="background-color: rgb(219, 168, 231);">
+
+    <p style="background-color: tomato;color:white ;">
+        Цей текст повинен бути відредагований за допомогою вбудованих стилів
+
+
+    </p>
+
+    <p style="background-color: white; color: brown; font-size: 70px;text-align:center;">

```

Comment:
replace this style to `head`

in `head` you can add the tag `styles`
add class to this element and then style from global styles

```
<styles>
   .class {
      background-color: white;
      color: brown;
      font-size: 70px;
      text-align: center;
    }
</styles>
```